### PR TITLE
bird2: bump to version 2.15.1

### PR DESCRIPTION
**Pull request by bradfordzhang**
Signed-off-by: Bradford Zhang <zyc@zyc.name>
(cherry picked from commit a55a0d13f7ec0e79f9340d90d74ed312ff6c26ce)

Maintainer: @tohojo 
Compile tested: x86-64 generic 23.05
Run tested: x86-64 generic 23.05 test done
Description:

### CODE REVIEW

File: bird2/Makefile [hunk 1]
```diff
@@ -7,12 +7,20 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=bird2
+<<<<<<< HEAD
 PKG_VERSION:=2.13.1
+=======
+PKG_VERSION:=2.15.1
+>>>>>>> a55a0d1 (bird2: bump to version 2.15.1)
```
Comment (BKPepe) [inline bird2/Makefile:14]: It seems that merge conflict was not properly resolved. Would you mind to take a look, please? :)

Comment (bradfordzhang) [reply bird2/Makefile]: Sorry for my mistake… It's fixed now

**Comment by bradfordzhang** [comment]
@BKPepe This is tested too. Thanks! :D

**Comment by BKPepe** [comment]
Your PR template is not filled, no indication of run testing and compile tested based on the details, which we do have currently available from your side.  CI/CD does not do run testing for this package, yet. Support could be added.

**Pull request closed without merging**